flumeDevice

```diff
@@ -73,8 +74,8 @@ preferences
     }
     section
     {
-        input "flowAlarmTrigger", "number", title: "Count of consecutive flow readings to trigger alarm (0 to disable).  Note: one reading per refresh interval when flow is detected.", defaultValue: 45, required: true
-        input "flowAlarmLookback", "number", title: "How many minutes to look back to identify a consecutive flow reading?", defaultValue: 5, required: true
+        input "flowAlarmTriggerTime", "number", title: "Number of minutes where flow is observed anytime within the Flow Lookback to trigger alert (0 to disable).", defaultValue: 90, required: true
```

Comment:
Im setting this value for myself to 40m .. This is sort of knowing your system. 
However we defined 'flow' if you have a washing machine that runs longer that your alertTime .. it'll go off